### ISSUE TITLE
initialize_target_repos still referenced

### ISSUE THREAD

**Issue by mvidner** [agama-project/agama#1369]
Reading Agama::Software::Manager code I found a method call that is not defined.
https://github.com/openSUSE/agama/blob/c9a68ce2441a2b73e2af6feae0ee70968e7dca92/service/lib/agama/software/manager.rb#L338
It turns out it is a bug in a recent refactoring, #1329.

**Comment by mvidner** [comment]
The fix is trivial, the method was renamed to `initialize_target` to better reflect what it does.
Should we have tests covering this?

**Comment by imobachgs** [comment]
Good catch. It looks like it is related to the registration. And, yes, I think it might be a good idea to have at least a minimal test.

Let's plan to get this fixed in milestone 10.

**Comment by imobachgs** [comment]
This problem was fixed a long time ago. I am closing the bug report.
